docs/source/conf.py

```diff
@@ -37,7 +37,7 @@
 extensions = [
     'sphinx.ext.todo',
     'sphinx_markdown_tables',
- #   'recommonmark',
+    'recommonmark',  # Uncomment this line
```

Comment:
is it possible to achieve your objective, which isn't clear because there are no descriptions in this PR, without using the recommonmark?
https://github.com/readthedocs/recommonmark says it is now deprecated